app/Http/Controllers/ChargeController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Controllers;
+
+
+use Illuminate\View\View;
+
+class ChargeController extends Controller
+{
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return response()->make();
+    }
+
+    public function index() : View {
+        $data = session('data');
+
+        return view('charges')->with('data', $data ? : null);
```

Comment:
`session(...)` returnere allerede `null` som default.

Så er det ikke nok bare at returnere session(..) direkte? -- eller er det for at være explicit ? 
```php
return view('charges')->with('data', session('data'));
```

Reply:
Tjek, godt set :) Vi reducerer